### PR TITLE
SMHE-1699: Do not throw MAX_SCHEDULE_TIME_EXCEEDED error, but original error

### PR DESCRIPTION
**Pull request by kroesctrl**
SMHE-1699: Do not throw max_retry or MAX_SCHEDULE_TIME_EXCEEDED error, but original error

### CODE REVIEW

**Comment by kroesctrl** [comment]
https://jenkins.fdp.osgp.cloud/view/SMHE-cucumber/job/gxf-smhe-cucumber-build/213/
https://jenkins.fdp.osgp.cloud/view/SMHE-cucumber/job/gxf-apps-smart-metering-integration-smhe-cucumber-build/902/

File: osgp/platform/osgp-core/src/main/java/org/opensmartgridplatform/core/application/services/DeviceResponseMessageService.java [hunk 1]
```diff
@@ -325,13 +325,29 @@ private void handleMaxScheduleTimeExceeded(
...
-            .osgpException(maxScheduleTimeExceededException)
+            .osgpException(originalOsgpException)
             .dataObject(message.getDataObject())
             .build();
     this.domainResponseMessageSender.send(maxScheduleTimeExceededResponseMessage);
   }
+
+  private FunctionalException getOriginalOsgpException(
+      final ProtocolResponseMessage message,
+      final FunctionalException maxScheduleTimeExceededException) {
```
Comment (harrymiddelburgetall) [inline osgp/platform/osgp-core/src/main/java/org/opensmartgridplatform/core/application/services/DeviceResponseMessageService.java]: parameterName suggest that exception is of type MAX_SCHEDULE_TIME_EXCEEDED. This still has to be checked inside this method. Use more generic name

**Review by harrymiddelburgetall — CHANGES_REQUESTED** [review]
One minor remark

File: osgp/platform/osgp-core/src/main/java/org/opensmartgridplatform/core/application/services/DeviceResponseMessageService.java (hunk 1 above)
Comment (kroesctrl) [reply osgp/platform/osgp-core/src/main/java/org/opensmartgridplatform/core/application/services/DeviceResponseMessageService.java]: Changed to functionalException

**Review by harrymiddelburgetall — APPROVED** [review]

**Pull request merged**